typings/vega-lite.d.ts

```diff
@@ -1,4 +1,3 @@
-declare module vl {
-  export var version: string;
-  export function compile(inputSpec: any, logger?: any): any;
-}
+export as namespace vl;
```

Comment:
This file is weird.  It seems a bit recursive.  but I guess we need it for docs & gallery.